stan/math/prim/fun/to_complex.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef STAN_MATH_PRIM_FUN_TO_COMPLEX_HPP
+#define STAN_MATH_PRIM_FUN_TO_COMPLEX_HPP
+
+#include <stan/math/prim/meta.hpp>
+#include <type_traits>
+#include <complex>
+
+namespace stan {
+namespace math {
+
+/**
+ * Return a complex type from a real part and an imaginary part.
+ * Default values for both parts is 0.
+ *
+ * @tparam T type of real and
```

Comment:
These don't need "and" (though that is how you'd write mathematics).  And it should be "type of real component".  The "type of real" would be `double` or `float`.